src/signing/signing.controller.ts

```diff
@@ -0,0 +1,47 @@
+import { Controller, Get, Param } from '@nestjs/common';
+import {
+  ApiBadRequestResponse,
+  ApiNotFoundResponse,
+  ApiOkResponse,
+  ApiOperation,
+  ApiParam,
+  ApiTags,
+} from '@nestjs/swagger';
+
+import { SignerDetailsDto } from '~/signing/dto/signer-details.dto';
+import { SignerModel } from '~/signing/models/signer.model';
+import { SigningService } from '~/signing/signing.service';
+
+@ApiTags('signer')
+@Controller('signer')
+export class SigningController {
+  constructor(private readonly signingService: SigningService) {}
+
+  @ApiOperation({
+    summary: 'Fetch signer details',
+    description: 'This endpoint provides information associated with a particular `signer`',
+  })
+  @ApiParam({
+    name: 'signer',
+    description:
+      'The value of the `signer` to fetch the address for. Note, the format depends on the signing manager the API is configured with',
+    type: 'string',
+    example: 'alice',
+  })
```

Comment:
Since you have described the param here, you can remove the @ApiProperty from `SignerDetailsDto`